lib/modules/betteReddit.js

```diff
@@ -167,19 +162,6 @@ module.go = function() {
 		}
 	}
 
-	if (module.options.scoreHiddenTimeLeft.value && isPageType('comments')) {
-		$('.tagline').on('mouseenter', 'span:contains([)', function() {
```

Comment:
I see why this is super inefficient, but I'm not sure that we should just remove it.

I think the usual pattern for doing stuff to a bunch of comments:

```js
applyScoreHiddenTimeLeft();
watchForElement(...);
// ...
function applyScoreHiddenTimeLeft() {
    querySelectorAll('.score-hidden')::forEachChunked(...)
}
```

would be a viable replacement. Might as well run it in afterLoad, since it's not noticeable until you hover.